bundles/config/org.eclipse.smarthome.config.xml.test/src/test/resources/test-bundle-pool/yahooweather.bundle/ESH-INF/i18n/yahooweather_de.properties

```diff
@@ -6,4 +6,7 @@ config.config.Dummy.question.option.667931 = K
 config.config.Dummy.unit.label = Einheit
 config.config.Dummy.unit.description = Spezifiziert die Einheit der Daten. Valide Werte sind 'us' und 'metric'
 config.config.Dummy.refresh.label = Aktualisierungsintervall
-config.config.Dummy.refresh.description = Spezifiziert das Aktualisierungsintervall in Sekunden
\ No newline at end of file
+config.config.Dummy.refresh.description = Spezifiziert das Aktualisierungsintervall in Sekunden
+
+config.configGroup.Dummy.group1.label = Group 1 German Label
+config.configGroup.Dummy.group1.description = Group 1 German Description

```

Comment:
Regarding to my previous comment I would change it to `config.config.Dummy.group1.description`